festim/hydrogen_transport_problem.py

```diff
@@ -164,6 +167,10 @@ def temperature_time_dependent(self):
     def multispecies(self):
         return len(self.species) > 1
 
+    @property
+    def volume_subdomains(self):
+        return [s for s in self.subdomains if isinstance(s, F.VolumeSubdomain)]
+
```

Comment:
See my comment below, I think we can keep both `volume_subdomains` and `surface_subdomains` but instead adapt the arguments of `Mesh1D.define_meshtags`